python/lsst/pipe/tasks/drpAssociationPipe.py

```diff
@@ -52,20 +55,66 @@ class DrpAssociationPipeConnections(pipeBase.PipelineTaskConnections,
         deferLoad=True,
         multiple=True
     )
+    ssObjectTableRefs = pipeBase.connectionTypes.Input(
+        doc="Reference to catalogs of SolarSolarSystem objects expected to be "
+            "observable in each (visit, detector).",
+        name="preloaded_DRP_SsObjects",
+        storageClass="ArrowAstropy",
+        dimensions=("instrument", "visit", "detector"),
+        minimum=0,
```

Comment:
I like this solution, if it means what I think it means; if you run this task, with `doSolarSystemAssociation=True`, but in a repo with no ephemerides for some or all of the visits, it just skips doing the solar system associations, and does regular DIAObject association only?  Does it warn?